renderEngine/Command/Collection/repeaterCommand.js

```diff
@@ -0,0 +1,50 @@
+import CommandUtil from "../../../test/command/CommandUtil.js";
+import IContext from "../../Context/IContext.js";
+import GroupResult from "../../Models/GroupResult.js";
+import SourceCommand from "../Source/BaseClasses/SourceCommand.js";
+import CallCommand from "./CallCommand.js";
+import SourceBaseCommand from "../SourceBaseCommand.js";
+import CommandBase from "../CommandBase.js";
+import JsonSource from "../../Source/JsonSource.js";
+import ICommandResult from "../../Models/ICommandResult.js";
+import LocalContext from "./../../Context/LocalContext.js";
+export default class RepeaterCommand extends SourceBaseCommand {
```

Comment:
Please change base class to `GroupCommnad` and move `.js` file to appropriate location.
Then edit ctor.